tools/plugins/com.liferay.ide.core/src/com/liferay/ide/core/IWorkspaceProject.java

```diff
@@ -26,13 +26,13 @@
  */
 public interface IWorkspaceProject extends ILiferayProject {
 
-	public List<IProject> getChildProjects();
+	public Set<IProject> getChildProjects();
 
 	public List<IPath> getTargetPlatformArtifacts();
 
 	public boolean isWatchable();
 
-	public void watch(Set<IProject> childProjects);
+	public void watch(Set<IProject> childProjects, IPath liferayHome);
```

Comment:
a workspace project should not need liferayHome.  it is configured from within workspace.  workspace knows the liferay home.